### PR TITLE
Update README.md to Specify `load` Statements

### PR DESCRIPTION
**Pull request by abhillman**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] ~Tests for the changes have been added (for bug fixes / features)~ not applicable
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://github.com/bazelbuild/rules_docker/issues/2166

## What is the new behavior?

Show `load` statements above rules usage in `README.md`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

This view is helpful to visualize the changes https://github.com/bazelbuild/rules_docker/pull/2167/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

### CODE REVIEW

**Comment by abhillman** [comment]
cc/ @uhthomas

File: README.md [hunk 1]
```diff
@@ -1128,6 +1147,8 @@ to use container_pull with custom docker authentication credentials.
 This target pushes on `bazel run :push_foo`:
 
 ``` python
+load("//container:container.bzl", "container_push")
```
Comment (thesayyn) [inline README.md:1150]:
```suggestion
load("@io_bazel_rules_docker//container:container.bzl", "container_push")
```

File: README.md [hunk 2]
```diff
@@ -1091,6 +1103,8 @@ and see <a href=#go_image-custom-base>go_image (custom base)</a> example.
 ### container_bundle
 
 ```python
+load("//container:container.bzl", "container_bundle")
```
Comment (thesayyn) [inline README.md:1106]:
```suggestion
load("@io_bazel_rules_docker//container:container.bzl", "container_bundle")
```

File: README.md [hunk 3]
```diff
@@ -335,6 +339,8 @@ That flag is typically passed in the `.bazelrc` file, see for example [`.bazelrc
 The second option is to employ `Makefile`-style variables:
 
 ```python
+load("//container:container.bzl", "container_bundle")
```
Comment (thesayyn) [inline README.md:342]:
```suggestion
load("@io_bazel_rules_docker//container:container.bzl", "container_bundle")
```

File: README.md [hunk 4]
```diff
@@ -296,6 +298,8 @@ A python format placeholder (e.g. `{BUILD_USER}`)
 is replaced by the value of the corresponding workspace-status variable.
 
 ```python
+load("//container:container.bzl", "container_push")
```
Comment (thesayyn) [inline README.md:301]:
```suggestion
load("@io_bazel_rules_docker//container:container.bzl", "container_push")
```

**Review by uhthomas — APPROVED** [review]

**Review by SarthakC — APPROVED** [review]

**Pull request closed without merging**